### ISSUE TITLE
Nicely done; could use doc strings

### ISSUE THREAD

**Issue by rswgnu** [eLvErDe/asyncio_process_watcher#1]
Nice, easy to read and follow code.  It could use some module and routine doc strings but is well done, compact and useful.  Thanks for sharing it.

**Comment by eLvErDe** [comment]
Happy to see it helps :-) it has been written in a hurry after being asked "we have this important process crashing on windows, do you have an idea?", that's why it is a bit hacky and not documented ;-)

**Comment by rswgnu** [comment]
Of course.  If you get the time, it would be good since it is now public.
